### ISSUE TITLE
Long links

### ISSUE THREAD

**Issue by cup** [commonmark/commonmark-spec#631]
Long links

Sometimes I will have to deal with a long link, like this:

https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String/lastIndexOf

I prefer to wrap long links if possible. This is allowed with reStructuredText:

~~~
lastIndexOf_

.. _lastIndexOf: https://developer.mozilla.org/docs/Web/JavaScript/
   Reference/Global_Objects/String/lastIndexOf
~~~

and also with AsciiDoc via link syntax:

~~~
link:++https://developer.mozilla.org/docs/Web/JavaScript/Reference/
Global_Objects/String/lastIndexOf++[lastIndexOf]
~~~

or variable syntax:

~~~
:js: https://developer.mozilla.org/docs/Web/JavaScript/Reference

{js}/Global_Objects/String/lastIndexOf
~~~

Does MarkDown have some way to deal with this?

- https://asciidoctor.org/docs/asciidoc-syntax-quick-reference#links
- https://asciidoctor.org/docs/user-manual#attributes
- https://docutils.sourceforge.io/docs/user/rst/quickref.html#hyperlink-targets

**Comment by johelegp** [comment]
> Does MarkDown have some way to deal with this?

No. Neither autolinks nor link destinations in inline links and link references allow newlines.
